apps/studio/components/interfaces/Database/Roles/RoleRow.tsx

```diff
@@ -200,7 +201,7 @@ const RoleRow = ({ role, disabled = false, onSelectDelete }: RoleRowProps) => {
                               </Tooltip.Content>
                             </Tooltip.Portal>
                           </Tooltip.Root>
-                        )
+                        ) as any)
```

Comment:
@ivasilov curious about what to do here. 
I typed role permissions which caused this to cascade. 

afterLabel is typed as string now, but as you can see, it is not that.

Reply:
afterLabel is meant to be a string (currently) so no wonder this doesn't work.

We're going to deprecate these components anyway, and then you won't have props like `afterLabel` anymore.

Reply:
https://github.com/supabase/supabase/blob/master/packages/ui/src/components/Toggle/Toggle.tsx/#L14-L15

quick fix: maybe change these to `string | React.ReactNode`